src/usdt/mod.rs

```diff
@@ -0,0 +1,220 @@
+use std::{cell::RefCell, os::raw::c_char};
+use std::ffi::CStr;
+use std::path::Path;
+use std::ptr;
+
+use bcc_sys::bccapi::{
+    bcc_usdt_close, bcc_usdt_enable_fully_specified_probe, bcc_usdt_enable_probe,
+    bcc_usdt_foreach_uprobe, bcc_usdt_genargs, bcc_usdt_new_frompath, bcc_usdt_new_frompid, pid_t,
+};
+use std::os::raw::c_int;
```

Comment:
This should be up with the rest from std - I like to keep std, external crate, and local crate use statements in their own groups.